articles/azure/azs-how-manage-azure-group.md

```diff
@@ -11,7 +11,11 @@ toc_sub3:
 toc_sub4:
 toc_title: Manage Azure user groups and permissions 
 toc_fullpath: Users/How To/azs-how-manage-azure-group.md
+<<<<<<< HEAD:articles/azure/azs-how-manage-azure-group.md
```

Comment:
What's the <<<<< HEAD bit for?

Reply:
These were automatically generated by github when we were merging different branches - however if you review the latest version of the document it shouldn't be present